infrastructure/aat.tfvars

```diff
@@ -0,0 +1 @@
+redis_capacity = "3"
```

Comment:
Are you sure about this change?

Reply:
@udaydenduluri33 - It got automatically added while building

Reply:
It was added by the PET team and pushed to master branch on 8th March

Reply:
Can you rebase it or remove it? THanks